### ISSUE TITLE
SyntaxError: expected ':'  in python/lib/ptxprint/xdv/colouring.py

### ISSUE THREAD

**Issue by msavli** [sillsdev/ptx2pdf#930]
SyntaxError: expected ':'  in python/lib/ptxprint/xdv/colouring.py

`python3-ptxprint is already the newest version (2.4.4-0~202311171045~ubuntu22.04.1).
0 nadgrajenih, 0 na novo nameščenih, 0 bo odstranjenih in 9 ne nadgrajenih.
1 ne popolnoma nameščenih ali odstranjenih.
Po tem opravilu bo porabljenega 0 B dodatnega prostora.
Ali želite nadaljevati? [Y/n] 
Setting up python3-ptxprint (2.4.4-0~202311171045~ubuntu22.04.1) ...
  File "/usr/lib/python3/dist-packages/ptxprint/xdv/colouring.py", line 35
    def font(self, opcode, fontnum)
                                   ^
SyntaxError: expected ':'
dpkg: error processing package python3-ptxprint (--configure):
 installed python3-ptxprint package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 python3-ptxprint
E: Sub-process /usr/bin/dpkg returned an error code (1)
`

pc@pc:~/Dokumenti/osis2usfm$ uname -a
Linux pc 6.2.0-35-generic #35~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Oct  6 10:23:26 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
pc@pc:~/Dokumenti/osis2usfm$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy

**Comment by n8marti** [comment]
After manually fixing the above error I came across more:
```
Paramétrage de python3-ptxprint (2.4.4-0~202311171045~ubuntu20.04.1) ...
  File "/usr/lib/python3/dist-packages/ptxprint/xdv/colouring.py", line 61
    except ValueError, TypeError: # <-- needs parentheses around Error tuple; i.e. (ValueError, TypeError)
                     ^
SyntaxError: invalid syntax

dpkg: erreur de traitement du paquet python3-ptxprint (--configure) :
 installed python3-ptxprint package post-installation script subprocess returned
 error exit status 1
```
and a for-loop with nothing happening inside:
```
102         for grange in groupby(gorder, key=lambda x:colours[x]):
103             # colour groupings # <-- nothing happens in the for-loop; adding "pass" at least allows for pkg installation
104 
105         
106
107

```

**Comment by davidg-sil** [comment]
Is this fixed on the lastest auto-build? fc06d5a8965a6 looks relevant.
